CMakeLists.txt

```diff
@@ -148,4 +148,9 @@ install(FILES
           ${CMAKE_CURRENT_BINARY_DIR}/${CMAKE_SHARED_LIBRARY_PREFIX}XCTest${CMAKE_SHARED_LIBRARY_SUFFIX}
         DESTINATION
           ${CMAKE_INSTALL_FULL_LIBDIR}/swift/${swift_os})
-
+if(NOT CMAKE_SYSTEM_NAME STREQUAL Darwin)
```

Comment:
It looks like you're now installing two distinct copies of libXCTest.so onto the system here? Would it be possible to use a symlink instead?

Reply:
Do not merge or take time on this merge request for the moment. I have to review the approach.

There is two copies. This change was a change compatible with the current setup and the setup from apple/swift#19432, which moves the binary libraries into the architecture directories for platforms that do not support fat libraries (every platform except Darwin). The duplication allowed me to merge this while working in the other change, and keep things working.

The symlink approach would work for some platforms, but not others (Windows), so I decided for a simpler copy, since this was going to be a temporal solution.